GetStream/stream-chat-android
# [4666] Fix multiple substitutions specified in non-positional format

### 🎯 Goal

Closes #4666

### 🛠 Implementation details

Replaced non-positional placeholders with positional ones.

### 🎨 UI Changes

No UI changes, just gets rid of a build warning.

### 🧪 Testing

1. Switch phone/ emulator to Korean/ Japanese
2. Open up selected channel menu and confirm that the number of members and online members is the correct one compared to English
3. Open a channel and check the same for the message list header

### ☑️Contributor Checklist

#### General
- [x] PR targets the ~~`develop`~~ `v5` branch

#### Code & documentation

### ☑️Reviewer Checklist

### 🎉 GIF

![gif](https://giphy.com/gifs/sportseditor-skateboarding-skateboard-l0IulSRkjIYAaCHCM)
